src/components/toolbar/toolbar-export-button.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FaFileDownload as IconDownload } from 'react-icons/fa';
+import ColladaExporter from 'three/examples/js/exporters/ColladaExporter';
+import { xml2js, js2xml } from 'xml-js';
+
+import { saveAs } from 'file-saver';
+import ToolbarButton from './toolbar-button';
+import { Project } from '../../class/export';
+import { parseData } from '../viewer3d/scene-creator';
+
+export default function ToolbarExportButton(
+  { state, context, catalog },
+  { translator }
+) {
+  let saveProjectToFile = e => {
+    e.preventDefault();
+    state = Project.unselectAll(state).updatedState;
+    let actions = {
+      areaActions: context.areaActions,
+      holesActions: context.holesActions,
+      itemsActions: context.itemsActions,
+      linesActions: context.linesActions,
+      projectActions: context.projectActions
+    };
+
+    let scene = state.get('scene');
+    // LOAD DATA
+    let planData = parseData(scene, actions, catalog);
+
+    setTimeout(() => {
```

Comment:
This is not safe, it leads to bugs when the planData is not fully created (during my tests I was able to export an empty scene)